content/documentation/device-examples/LEDMatrix.md

```diff
@@ -0,0 +1,52 @@
+---
+title: LEDMatrix
+weight: 207
+tags: ["LEDMatrix"]
+---
+### Description
+The [LEDMatrix](https://github.com/Pi4J/pi4j-example-components/tree/Dev-Arcade/src/main/java/com/pi4j/example/components) (src/main/java/com/pi4j/example/components) is a template class, that you can use in your own Java-project.
+It is an extension of the class [LEDStrip](/documentation/device-examples/ledstrip). You can use many Neopixel-LED-Strips, cut them in between the LEDs, and sold them back together with wires.
```

Comment:
Instead of:
You can use many Neopixel-LED-Strips, cut them in between the LEDs, and sold them back together with wires.

Use:
An LED matrix can be built from one LED strip. To do this, separate the LED strip at the desired point and place the individual strips under each other or next to each other. The individual ends can then be connected to each other with a wire. 

The constructor can be passed either a rectangular matrix or a user-defined matrix with different numbers of LEDs in the individual strips.